src/screens/profile/index.tsx

```diff
@@ -1,12 +1,73 @@
 import React from 'react';
 import {Text, View} from 'react-native';
+import {Avatar} from 'native-base';
+import styles from './styles';
 
 const Profile = () => {
+  const DATA_MOCK = {
+    firstName: 'Ayom',
+    lastName: 'Aristo',
+    skill: 67,
+    position: 'Flex',
+    status: 'Starter',
+    fidelity: 100,
+    matchsPlayed: 1,
+  };
+
   return (
-    <View>
-      <Text>Profile SCREEN</Text>
+    <View style={styles.container}>
+      <Avatar
+        style={styles.avatarContainer}
+        bg="lightBlue.400"
+        source={{
+          uri: 'https://images.unsplash.com/photo-1603415526960-f7e0328c63b1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
+        }}
+        size="2xl">
+        NB
+        <Avatar.Badge
+          bg={DATA_MOCK.status === 'Starter' ? 'green.500' : 'red.500'}
+        />
+      </Avatar>
+      <View style={styles.infoContainer}>
+        <View style={styles.infoRow}>
```

Comment:
We can create a FlatList to not repeat echa `infoRow`, where each renderItem is going to return: `<View style={styles.infoRow}>
          <Text style={styles.infoRowTitle}>Last Name</Text>
          <Text style={styles.infoRowText}>{DATA_MOCK.lastName}</Text>
        </View>`

Reply:
Done.